frontend/src/pages/Errors/ErrorsPage.tsx

```diff
@@ -11,6 +11,5 @@ export const EmptyErrorsSearchParams: Complete<ErrorSearchParamsInput> = {
 	os: undefined,
 	visited_url: undefined,
 	type: undefined,
-	// TODO: remove the other attributes after deprecating static filters
```

Comment:
We still need to clean up the static filters

Reply:
we should cleanup all the errors / sessions searching - looks like a relatively large effort since the segments also serialize this large object. making a ticket to follow up on this: #4534